.rubocop.yml

```diff
@@ -54,7 +56,7 @@ Layout/EmptyLinesAroundMethodBody:
 Layout/EmptyLinesAroundModuleBody:
   Enabled: true
 
-Layout/FirstParameterIndentation:
+Layout/IndentFirstArgument:
```

Comment:
I think it is preferable to move in ASCII order.
Ref: https://github.com/rails/rails/pull/36426/files#r291066852

Reply:
Agreed.